StormyHares/BangazonApi
# Seed cm

## Ticket Link:
  - No Ticket

## Description:
  - Used code from Jason Figuroa to seed the database upon creation

## Steps to Test:

  - Git fetch and checkout into seed-cm
  -  Delte BangazonApi.db
  - From the BangazonApi directory, run the ```dotnet ef database update``` to create the database file
- Run ```dotnet run``` to compile and run the code
- Open the newly created BangazonApi.db in DB Browser for SQLite and confirm that data has been populated
